File: serverH/server/HostelOperation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace serverH
{
    public interface HostelOperation
    {
        List<Hostel> getListOfHostel();
        List<Hostel> addNewHostel(Hostel item);
        List<Hostel> delHostel(int index);
        int getSumOfHostel();
    }
}

File: Server/src/operation/HostelOperation.java
package operation;

import types.Hostel;

import java.util.List;

public interface HostelOperation {
    List<Hostel> getListOfHostel();
    List<Hostel> addNewHostel(Hostel hostel);
    List<Hostel> delHostel(int index);
    int getSumOfHostel();
}
